sycl/unittests/Extensions/CommandGraph/Exceptions.cpp

```diff
@@ -393,6 +393,27 @@ TEST_F(CommandGraphTest, EnqueueCustomCommandCheck) {
   ASSERT_EQ(ExceptionCode, sycl::errc::invalid);
 }
 
+// sycl_ext_oneapi_work_group_scratch_memory isn't supported with SYCL graphs
```

Comment:
Could you add an entry to this section of the SYCL-Graph spec https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc#712-interaction-with-other-extensions documenting that an exception will be thrown if a user tries to use `sycl_ext_oneapi_work_group_scratch_memory` in a graph.

Reply:
Sure, added entry in the last commit